eq-author/src/components/CollapsibleToggled/index.js

```diff
@@ -28,22 +27,24 @@ const CollapsibleToggled = ({
   defaultOpen = false,
```

Comment:
```suggestion
  isOpen = false,
```

If you do the change that renames this prop, this will need to change.